.eslintrc.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
```

Comment:
:heart: - please bring this up in the next dev catch-up if I forget? We should sync up on JS linting standards.